### PR TITLE
Show owner pubkey as a hex string in nft list command

### PR DESCRIPTION
**Pull request by AmineKhaldi**

### CODE REVIEW

**Review by xdustinface — CHANGES_REQUESTED** [review]
Since `owner_pubkey` is an `Optional` type, this will make `mypy` unhappy as soon as `from_json_dict` gets hinted properly like in https://github.com/Chia-Network/chia-blockchain/pull/11763/commits/f196520359cc4d083e7121220352de611ad847c4#diff-dd6f0c93f94df70ed7b9693a16d72d4d79e9f2338eb65b13bebad03b28d9593cR667

```
chia/cmds/wallet_funcs.py:773: error: Item "None" of "Optional[bytes]" has no attribute "hex"  [union-attr]
```

I guess you could add an `assert nft.owner_pubkey is not None` at the top because in this case it will probably always be set, right?

**Review by xdustinface — APPROVED** [review]
Looks good, but it would also fit into one line, see my suggestions :)

File: chia/cmds/wallet_funcs.py [hunk 1]
```diff
@@ -763,13 +763,17 @@ async def list_nfts(args: Dict, wallet_client: WalletRpcClient, fingerprint: int
...
+                else:
+                    owner_pubkey = nft.owner_pubkey.hex()
                 print()
                 print(f"{'Launcher coin ID:'.ljust(26)} {nft.launcher_id}")
                 print(f"{'Launcher puzhash:'.ljust(26)} {nft.launcher_puzhash}")
                 print(f"{'Current NFT coin ID:'.ljust(26)} {nft.nft_coin_id}")
                 print(f"{'On-chain data/info:'.ljust(26)} {nft.chain_info}")
                 print(f"{'Owner DID:'.ljust(26)} {nft.owner_did}")
-                print(f"{'Owner pubkey:'.ljust(26)} {nft.owner_pubkey}")
+                print(f"{'Owner pubkey:'.ljust(26)} {owner_pubkey}")
```
[the remark this answers is not in the document]
Comment (AmineKhaldi) [reply chia/cmds/wallet_funcs.py:776]: I tried it but that line got too long and it stood out, so I opted for the above. I don't mind either though so it's up to you guys. Thanks!

**Review by wjblanke — APPROVED** [review]
aok

**Pull request merged**